app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h3> Contestant Information</h3>
+<% @contestant.each do |c| %>
+  <h3> <%= c.name %></h3>
+  <h3>Projects: <%= c.projects.pluck(:name).join(', ') %></h3>
```

Comment:
It is not our views job to do data logic either. So you would want to create a method in your `Contestant` model to get the names. So something like:
```ruby
def projects_list
  projects.pluck(:name).join(', ')
end

Or 

def projects_list
  projects.pluck(:name).to_sentence
end
```